### ISSUE TITLE
Problema con "claveForánea"

### ISSUE THREAD

**Issue by aboglioli** [jafidiesel/MinervaModeling#1]
Estaba pensando que la referencia entre _Entidad_ y _Atributo_ llamada **claveForanea**, si bien está bien pensada, se nos va a complicar más adelante cómo saber a qué Entidad apunta esa clave foránea, o sea, ¿qué entidad tiene como clave primaria aquella a la que **claveForanea** hace referencia?

Es un problema ya que _Relacion_ es la única que conoce las entidades relacionadas entre sí; las entidades sólo conocen sus atributos.

**Comment by jafidiesel** [comment]
Estaba pensando en que si volvemos a la opcion original (la de Atributo con los dos boolean) aun así no vamos a saber si, por ejemplo, ese atributo que es clave foranea (claveForanea=True) no vamos a saber de que Entidad es clave primaria. 
Se entiende?

**Comment by aboglioli** [comment]
Claro, también está ese tema, ninguno de los dos enfoques sirve. Habría que trasladar el atributo de claveForánea a EntidadRelación o algo así, ya que es propio de una relación (esto que digo es a modo de ejemplo porque no he pensado bien si esto funcionaría).

Lo que no sé es cómo funcionaría EntidadRelación a la hora de que el usuario comience a usar la herramienta, porque este no va a tener un ícono para arrastrar y soltar un elemento de EntidadRelacion, sino que este elemento debe crearse automáticamente cuando el usuario _linkea_ mediante una relación dos o más entidades. ¿Se entiende?
